docs/OpenAPI/openapi.json

```diff
@@ -1 +1 @@
Response","content":{"application/json":{"schema":{}}}},"422":{"description":"Validation Error","content":{"application/json":{"schema":{"$ref":"#/components/schemas/HTTPValidationError"}}}}}}},"/v1/WMTSCapabilities.xml":{"get":{"tags":["OGC"],"summary":"Wtms","description":"Wmts endpoit.","operationId":"wtms_v1_WMTSCapabilities_xml_get","parameters":[{"description":"Cloud Optimized GeoTIFF URL.","required":true,"schema":{"title":"Url","type":"string","description":"Cloud Optimized GeoTIFF URL."},"name":"url","in":"query"},{"description":"Output image type. Default is png.","required":false,"schema":{"title":"Tile Format","enum":["png","npy","tif","jpg","webp"],"type":"string","description":"Output image type. Default is png.","default":"png"},"name":"tile_format","in":"query"},{"description":"Tile size scale. 1=256x256, 2=512x512...","required":false,"schema":{"title":"Tile Scale","exclusiveMaximum":4.0,"exclusiveMinimum":0.0,"type":"integer","description":"Tile size scale. 1=256x256, 2=512x512...","default":1},"name":"tile_scale","in":"query"}],"responses":{"200":{"description":"Successful Response","content":{"application/xml":{"schema":{"type":"string"}}}},"422":{"description":"Validation Error","content":{"application/json":{"schema":{"$ref":"#/components/schemas/HTTPValidationError"}}}}}}}},"components":{"schemas":{"HTTPValidationError":{"title":"HTTPValidationError","type":"object","properties":{"detail":{"title":"Detail","type":"array","items":{"$ref":"#/components/schemas/ValidationError"}}}},"TileJSON":{"title":"TileJSON","required":["tiles","center"],"type":"object","properties":{"tilejson":{"title":"Tilejson","type":"string","default":"2.2.0"},"name":{"title":"Name","type":"string"},"description":{"title":"Description","type":"string"},"version":{"title":"Version","type":"string","default":"1.0.0"},"attribution":{"title":"Attribution","type":"string"},"template":{"title":"Template","type":"string"},"legend":{"title":"Legend","type":"string"},"scheme":{"title":"Scheme","type":"string","default":"xyz"},"tiles":{"title":"Tiles","type":"array","items":{"type":"string"}},"grids":{"title":"Grids","type":"array","items":{"type":"string"}},"data":{"title":"Data","type":"array","items":{"type":"string"}},"minzoom":{"title":"Minzoom","maximum":30.0,"minimum":0.0,"type":"integer","default":0},"maxzoom":{"title":"Maxzoom","maximum":30.0,"minimum":0.0,"type":"integer","default":30},"bounds":{"title":"Bounds","type":"array","items":{"type":"number"},"default":[-180,-90,180,90]},"center":{"title":"Center","type":"array","items":[{"type":"number"},{"type":"number"},{"type":"integer"}]}},"description":"TileJSON model.\n\nBased on https://github.com/mapbox/tilejson-spec/tree/master/2.2.0"},"ValidationError":{"title":"ValidationError","required":["loc","msg","type"],"type":"object","properties":{"loc":{"title":"Location","type":"array","items":{"type":"string"}},"msg":{"title":"Message","type":"string"},"type":{"title":"Error Type","type":"string"}}}}}}
```

Comment:
we might want to pretty print format this so it doesn't appear as a giant diff when any changes are made.

Reply:
I think it's auto generated, but maybe there is a way to pretty print the autogenerated result.